### PR TITLE
EWPP-929: Drupal 9 compatibility fixes.

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["${selenium.browser}", null, "${selenium.host}:${selenium.port}/wd/hub"]'/>
   </php>
   <testsuites>
-    <testsuite>
+    <testsuite name="unit_tests">
```
Comment (yenyasinn) [inline phpunit.xml.dist]: We have to provide some name here.

File: composer.json [hunk 2]
```diff
@@ -6,27 +6,25 @@
...
+        "drush/drush": "~10.0",
         "nikic/php-parser": "^3.1.5",
         "openeuropa/behat-transformation-context" : "~0.1",
-        "openeuropa/code-review": "~1.5",
-        "openeuropa/drupal-core-require-dev": "^8.9.13",
-        "openeuropa/task-runner": "~1.0.0-beta5",
-        "phpunit/phpunit": "~6.0",
-        "symfony/dom-crawler": "~3.4"
+        "openeuropa/code-review": "dev-EWPP-372",
+        "openeuropa/drupal-core-require-dev": "^8.9.13 || ^9.1.6",
```
Comment (ademarco) [inline composer.json]: This needs to be the same as what's in `"require"`, we let composer resolve versions.
```suggestion
        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
```

File: composer.json [hunk 3]
```diff
@@ -6,27 +6,25 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
+        "drupal/core": "^8.9 || ^9.1",
```
Comment (ademarco) [inline composer.json]: As we cannot be sure that our sites can be updated from 8.9 to 9.1 directly, without passing by 9.0, we have to make sure that a component will not add such a constraint, unless it is actually needed.

Please use here:

```suggestion
        "drupal/core": "^8.9 || ^9",
```

File: tests/modules/rdf_skos_test/rdf_skos_test.info.yml [hunk 4]
```diff
@@ -3,6 +3,6 @@ type: module
 description: 'Test functionality for the RDF SKOS module.'
 package: Testing
 version: VERSION
-core: 8.x
+core_version_requirement: ^8.9 || ^9.1
```
Comment (ademarco) [inline tests/modules/rdf_skos_test/rdf_skos_test.info.yml]:
```suggestion
core_version_requirement: ^8.9 || ^9
```

**Review by ademarco — CHANGES_REQUESTED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by ademarco — DISMISSED** [review]

**Review by upchuk — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**